ntbs-service/Models/Entities/Alerts/AlertWithTbServiceForDisplay.cs

```diff
@@ -8,10 +8,13 @@ namespace ntbs_service.Models.Entities.Alerts
     public class AlertWithTbServiceForDisplay
     {
         public int AlertId { get; set; }
+        [Display(Name = "NTBS Id")]
         public int? NotificationId { get; set; }
         public DateTime CreationDate { get; set; }
         public bool NotDismissable  { get; set; }
+        [Display(Name = "Alert Date")]
```

Comment:
I think the previous version had "date" and "type" rather than "Date" and "Type". I don't think it really matters, but it's probably safer to stick with what was there already.